### PR TITLE
Added auditd serice, fixed containers restart, added security settings for docker engine

### PR DESCRIPTION
**Pull request by VladimirNovgorodov**
Installed and configured Auditd service.
Set daemon.json file in /etc/docker.
Set containers log to local /var/log/docker folder and to remote server ( still hosts.yml file for specifying remote log server protocol, ip, port ). 
Created additional bridge-compose.yml file for poabridge service.

### CODE REVIEW

File: bridge-nodejs/roles/servinstall/templates/poabridge.j2 [hunk 1]
```diff
@@ -30,44 +30,33 @@ do
...
-}
-
 start(){
     echo "Starting bridge.."
     cd $WORKDIR
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose down -v
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose rm -fv
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose up --detach
-    runbridge
+    sudo -u "{{ compose_service_user }}" VALIDATOR_ADDRESS=$vaddr VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey /usr/local/bin/docker-compose  down -v
```
Comment (akolotov) [inline bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Are `VALIDATOR_ADDRESS` and `VALIDATOR_ADDRESS_PRIVATE_KEY` really needed to shutdown containers?

The same question is for the action to remove the container in the next line.

File: bridge-nodejs/roles/servinstall/templates/poabridge.j2 [hunk 2]
```diff
@@ -30,44 +30,33 @@ do
...
+    sudo -u "{{ compose_service_user }}" VALIDATOR_ADDRESS=$vaddr VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey /usr/local/bin/docker-compose  rm -fv
+    sudo -u "{{ compose_service_user }}" VALIDATOR_ADDRESS=$vaddr VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey /usr/local/bin/docker-compose  up  --detach
 }
 
 stop(){
     echo "Stopping bridge.."
     cd $WORKDIR
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose down -v 
+    sudo -u "{{ compose_service_user }}" VALIDATOR_ADDRESS=$vaddr VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey /usr/local/bin/docker-compose  down -v
```
Comment (akolotov) [inline bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Are VALIDATOR_ADDRESS and VALIDATOR_ADDRESS_PRIVATE_KEY really needed to shutdown containers?

File: bridge-nodejs/roles/servinstall/templates/poabridge.j2 [hunk 3]
```diff
@@ -30,44 +30,33 @@ do
...
     sleep 2
 }
 
 status(){
     echo "Bridge status:"
     cd $WORKDIR
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose ps 
+    sudo -u "{{ compose_service_user }}" VALIDATOR_ADDRESS=$vaddr VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey /usr/local/bin/docker-compose  ps
```
Comment (akolotov) [inline bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Are VALIDATOR_ADDRESS and VALIDATOR_ADDRESS_PRIVATE_KEY really needed to get of containers' status?

File: bridge-nodejs/roles/servinstall/templates/poabridge.j2 [hunk 4]
```diff
@@ -30,44 +30,33 @@ do
...
 }
 
 rebuild(){
     echo "Rebuild bridge.."
     cd $WORKDIR
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose down -v
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose rm -fv
-    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose up --detach --force-recreate --no-deps --build
-    runbridge
+    sudo -u "{{ compose_service_user }}" VALIDATOR_ADDRESS=$vaddr VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey /usr/local/bin/docker-compose  down -v
```
Comment (akolotov) [inline bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Are VALIDATOR_ADDRESS and VALIDATOR_ADDRESS_PRIVATE_KEY really needed to shutdown containers?

The same question is for the action to remove the container in the next line.

**Comment by akolotov** [comment]
@VladimirNovgorodov could you share a forecast when the comments raised in this PR are addressed?
We need address issues found during the security audit on this week. Is it possible?

@phahulin, fyi

**Comment by ArseniiPetrovich** [comment]
It looks like `VALIDATOR_ADDRESS` variable should be used in all containers (not the `VALIDATOR_ADDRESS_PRIVATE_KEY`, which is for one container only)

**Pull request merged**